src/controllers/apps/chat-app/message.controllers.js

```diff
@@ -186,8 +186,8 @@ const deleteMessage = asyncHandler(async (req, res) => {
   // Check if user is the sender of the message
   if (message.sender.toString() !== req.user._id.toString()) {
     throw new ApiError(
-      401,
-      "You are not authorised to delete the message ,you are not the sender"
+      400,
+      "You are not the owner to delete the message ,you are not the sender"
```

Comment:
Fix the formatting there should be space after the comma and not before and the error code should be 403.

Reply:
Who's code is this yours

Reply:
yes i have written the additional code is something wrong